.github/workflows/npm-release.yml

```
@@ -0,0 +1,63 @@
+name: npm-release
```

Shouldn't the name readable? 

```suggestion
name: Npm Release
```

---

```suggestion
name: npm release
```

---

Capitalization only on first word of sentences and proper nouns. npm is lower case on its own website.